<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[the name offailedis failed]]>
    </line>
    <line>
      <![CDATA[the name ofhomePageis failed]]>
    </line>
  </reporter-output>
  <suite started-at="2023-01-25T18:04:15 IST" name="Default suite" finished-at="2023-01-25T18:04:26 IST" duration-ms="10173">
    <groups>
    </groups>
    <test started-at="2023-01-25T18:04:15 IST" name="Default test" finished-at="2023-01-25T18:04:26 IST" duration-ms="10173">
      <class name="takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin">
        <test-method signature="failed(java.lang.String)[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:15 IST" name="failed" finished-at="2023-01-25T18:04:15 IST" duration-ms="0" status="FAIL">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Cannot inject @Test annotated Method [failed] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [failed] with [class java.lang.String].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:439)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.handleParameters(Parameters.java:884)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failed -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:15 IST" name="setUp" finished-at="2023-01-25T18:04:18 IST" duration-ms="2107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="initialization()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:18 IST" name="initialization" finished-at="2023-01-25T18:04:19 IST" duration-ms="1254" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:19 IST" name="teardown" finished-at="2023-01-25T18:04:19 IST" duration-ms="278" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:19 IST" name="setUp" finished-at="2023-01-25T18:04:20 IST" duration-ms="1292" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginPage()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:20 IST" name="loginPage" finished-at="2023-01-25T18:04:21 IST" duration-ms="846" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:21 IST" name="teardown" finished-at="2023-01-25T18:04:21 IST" duration-ms="264" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:21 IST" name="setUp" finished-at="2023-01-25T18:04:23 IST" duration-ms="1304" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="homePage()[pri:1, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:23 IST" name="homePage" finished-at="2023-01-25T18:04:25 IST" duration-ms="2289" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [actiTIME - nter Time-Track] but found [actiTIME - Enter Time-Track]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [actiTIME - nter Time-Track] but found [actiTIME - Enter Time-Track]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin.homePage(ActitimeTestNGLogin.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[the name ofhomePageis failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- homePage -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin@53976f5c]" started-at="2023-01-25T18:04:25 IST" name="teardown" finished-at="2023-01-25T18:04:26 IST" duration-ms="205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- takeScreenShotDynamicallyTestNG.ActitimeTestNGLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
